documentation.html

```diff
@@ -31901,16 +31940,15 @@ <h4 id="_creating_a_kar_file"><a class="anchor" href="#_creating_a_kar_file"></a
                     same feature twice.
                     Refer to Karaf forum posting at http://karaf.922171.n3.nabble.com/Duplicate-feature-repository-entry-using-archive-kar-to-build-deployable-applications-td3650850.html
                     --&gt;</span>
-                    <span class="tag">&lt;resourcesDir&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/doesNotExist<span class="tag">&lt;/resourcesDir&gt;</span>
-
+                    
                     <span class="comment">&lt;!--
-                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.1, it will need to be
+                    Location of the features.xml file. If it references properties that need to be filtered, e.g., 2.9.2, it will need to be
                     filtered by the maven-resources-plugin.
                     --&gt;</span>
-                    <span class="tag">&lt;featuresFile&gt;</span>/Users/phillip/Documents/workspace/release/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
+                    <span class="tag">&lt;featuresFile&gt;</span>/Users/smorris/ddf/ddf/distribution/docs/target/classes/features.xml<span class="tag">&lt;/featuresFile&gt;</span>
```

Comment:
There's a `{basedir}` in a code sample that asciidoctor apparently has a default variable for. I changed the syntax in one of my PRs to stop it happening, but I'll have to delete it manually from these ones that are already generated.